### DISCUSSION TITLE
Is it possible to know the path of error

### DISCUSSION THREAD

**Discussion by Freed-Wu** [python-jsonschema/jsonschema#1179] [Q&A] [answered]
Assume `Invalid` will trigger error when `validate()`:

```json
{"name" : "a", "attr": {"price" : "Invalid", "number": 20}}
```

Is it possible to know the path of error like `.attr.price`? TIA!

**Comment by Julian** [comment, marked as answer]
Yes, that is called the "path" of the error, and is available on any `ValidationError` object. See [here](https://python-jsonschema.readthedocs.io/en/stable/errors/#jsonschema.exceptions.ValidationError.absolute_path) for details.
